app/src/main/java/ru/prometeydev/movie/ui/moviesdetails/MoviesDetailsViewModel.kt

```diff
@@ -2,24 +2,23 @@ package ru.prometeydev.movie.ui.moviesdetails
 
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
-import androidx.lifecycle.ViewModel
-import androidx.lifecycle.viewModelScope
-import kotlinx.coroutines.launch
-import ru.prometeydev.movie.data.Movie
-import ru.prometeydev.movie.data.MoviesRepository
+import ru.prometeydev.movie.model.local.MovieDetails
+import ru.prometeydev.movie.model.MoviesRepository
+import ru.prometeydev.movie.ui.base.BaseViewModel
+import ru.prometeydev.movie.ui.base.Event
 
 class MoviesDetailsViewModel(
     private val repository: MoviesRepository
-) : ViewModel() {
+) : BaseViewModel() {
 
-    private val _mutableMovieState = MutableLiveData<Movie>()
+    private val _mutableLiveData = MutableLiveData<Event<MovieDetails>>()
```

Comment:
Duplication - you can use generics and move mutable live data and live data in the base model